report-extensions.js

```diff
@@ -143,61 +194,85 @@ function sortedKeys(s) {
 
     let content = summary;
     if (outdated) {
-        if (process.env.VALIDATE_PR !== 'true') {
-            process.exitCode = -1;
```

Comment:
why don't we fail anymore?

it seems that not all extensions are up to date, but job is green one, it looks misleading

Reply:
Thanks for bringing this up, @akosyakov. I see I forgot to mention it in the PR description, but the change was made because of the fact that the automatic job is always failing, and likely we will never hit green with all extensions, and even if we do, I would say it wouldn't last long. This is why now it is green by default, and fails if a big extension(s) that we have fails to publish (sum of all failing must be 5% of all total install counts). This means we can be alerted by a failing job about something going wrong which should always be prioritized (doesn't fail if an extension with 5 downloads fails to publish).

I understand that it looks misleading, but I believe that it's way more helpful than always seeing a red status indicator every time. I too recognize that it would be ideal if we fixed all of the extensions here but I do not think this is realistic currently.